### DISCUSSION TITLE
Prerequisites to configure steamguard-cli

### DISCUSSION THREAD

**Discussion by tupo2** [dyc3/steamguard-cli#309] [Q&A] [answered]
Can you tell me if these are needed to configure steamguard-cli:
- A phone number?
- The Steam app on phone?

**Comment by dyc3** [comment, marked as answer]
Yes, you need a phone number that can receive SMS. The setup process will guide you through it.

No, you don't need the steam app on your phone.

**Reply by tupo2** [reply]
OK, thanks.
